### PR TITLE
Fix recursive deletion of directories

### PR DESCRIPTION
**Pull request by SlimaneAmar**

### CODE REVIEW

**Review by AbdelHedhili — APPROVED** [review]

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+spring:
```
Comment (JacquesBorsenberger) [inline src/main/resources/application.yml:1]: why ? 
and why debug ?

**Review by JacquesBorsenberger — CHANGES_REQUESTED** [review]
I thing there is a configuration file that was not meant to be there, even if it is, loglevel to debug is a bit too much

File: src/main/resources/application.yml (hunk 1 above)
Comment (SlimaneAmar) [reply src/main/resources/application.yml]: oh sorry
it'a mistake

Comment (SlimaneAmar) [reply src/main/resources/application.yml]: this file is not in the commit

Comment (SlimaneAmar) [reply src/main/resources/application.yml]: removed

**Review by JacquesBorsenberger — APPROVED** [review]

**Pull request merged**